src/index.tsx

```diff
@@ -1,17 +1,20 @@
-import React from "react";
+import { StrictMode, createElement } from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 
 ReactDOM.render(
-  <React.StrictMode>
+  <StrictMode>
     <App />
-  </React.StrictMode>,
+  </StrictMode>,
   document.getElementById("root")
 );
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
+// HACK for https://github.com/dai-shi/excalidraw-claymate/issues/41
+(window as any).React = { createElement };
```

Comment:
This is probably only necessary to fix #41, but let's keep others as various things are updated.

Reply:
I understand all the changes except this one. What does this do?

Reply:
Without this, #41 bug persists. Somehow, `@excalidraw/excalidraw` package can't find `React.createElement` in the production build.
If you find another solution, much appreciated.

Reply:
Are you able to reproduce #41 locally?  I built claymate and ran it locally but can't reproduce the problem on master

Reply:
Yes, this is reproducible for me locally.

Reply:
I still can't reproduce the problem and the window element never seems to have a React property so I can't really add anything useful to this review.

Reply:
really... does the problem repro in the production for you?
(I don't like this hack and wish we have a proper fix.)

Reply:
No, I presume all I need to do is:
Open: https://dai-shi.github.io/excalidraw-claymate/
Click: "Export Gif"
Enter a file name & click "Save"

Reply:
![wBV7l3zsLH](https://user-images.githubusercontent.com/804371/132181274-04922fcd-dd37-4d67-8256-774ce81f3f00.gif)

Reply:
No, it's "Export" in excalidraw.
<img width="216" alt="image" src="https://user-images.githubusercontent.com/490574/132181256-6ac87f37-a834-4333-b6b6-19cfe5373bb5.png">

Reply:
Ok, now I see the problem!!!